src/lib.rs

```diff
@@ -185,7 +185,7 @@ pub fn init_logging(opentelemetry_url: &Option<Url>) -> Result<(), LemmyError> {
     .trim_matches('"')
     .parse::<Targets>()?;
 
-  let format_layer = tracing_subscriber::fmt::layer().with_filter(targets.clone());
+  let format_layer = tracing_subscriber::fmt::layer().with_filter(targets);
```

Comment:
Same issue as [the other PR](https://github.com/LemmyNet/lemmy/pull/3385#discussion_r1244434439) I made.